File: codeforces/4A.c
#include <stdio.h>

int main(int argc, char const *argv[])
{
    int n;
    scanf("%d", &n);
    if (n > 2)
    {
        printf("%s\n", ((n & 1) == 0)?"YES":"NO");    
    }
    else
    {
        printf("NO\n");
    }
    
    return 0;
}

File: codeforces/4A/4A.go
/*
 *http://codeforces.com/contest/4/problem/A
 */

package main

import (
	"fmt"
)

func watermelon(w int) string {
	if w%2 == 0 && w != 2 {
		return "YES"
	}

	return "NO"
}

func main() {
	for {
		var w int
		rc, err := fmt.Scanf("%d", &w)
		if rc < 0 || err != nil {
			break
		}
		fmt.Printf("%s\n", watermelon(w))
	}
}
